frontend/markdown-pdf/build.js

```diff
@@ -58,6 +62,45 @@ const markdownitRenderer = new markdownit({
     .use(require('../markdown-it-plugins/video-thumb'));
 
 const args = process.argv.slice(2);
+
+var languages = new Map();
+
+// Define a mapping between language codes and translation of 'Manual'
+const manual_translations = {
+    en: 'Manual',
+    es: 'Manual',
+    pt: 'Manual',
+    de: 'Anleitung',
+    fr: 'Manuel',
+    it: 'Manuale',
+    nl: 'Handleiding',
+    se: 'Anvandarhandbok',
+    fi: 'Ohjeet',
+    cz: 'Manuál',
+    ro: 'Instrucțiuni',
+    tr: 'Talimatlar',
+};
+
+// Define the ordering of the pages
+const ordering = [
+    'en',
+    'es',
+    'pt',
+    'de',
+    'fr',
+    'it',
+    'nl',
+    'se',
+    'fi',
+    'cz',
+    'ro',
+    'tr',
+];
+
+var DOCS_BASE_URL = process.env.DOCS_BASE_URL
+    ? process.env.DOCS_BASE_URL
+    : 'https://docs.victronenergy.com/';
```

Comment:
De evironment variables worden momenteel niet geladen dmv `dotenv`; en ook in de Dockerfile wordt de betreffende variable niet doorgeven.

Kunnen we de in plaats van stil een default gebruiken (welk voor sommige omgevingen helemaal niet zal kloppen) een warning gooien?